### PR TITLE
accounts: create/edit user form fix

### PR DESCRIPTION
**Pull request by diegodelemos**
* Makes available just email and active fields.

Signed-off-by: Diego Rodriguez <diego.rodriguez.rodriguez@cern.ch>

### CODE REVIEW

File: tests/test_admin.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 from flask_admin import Admin, menu
 from invenio_admin import InvenioAdmin
 from invenio_db import db
+from six import b
```
Comment (jirikuncar) [inline tests/test_admin.py]: do you need this import?

File: tests/test_admin.py [hunk 2]
```diff
@@ -92,3 +93,27 @@ def test_admin(app):
             data={'rowid': -42, 'action': 'activate'},
             follow_redirects=True
         )
+
+        # Test empty mail form
+
+        res = client.post(
+            url_for("user.create_view"),
```
Comment (jirikuncar) [inline tests/test_admin.py]: Please use preferably single quotes if possible. Thanks

File: tests/test_admin.py [hunk 3]
```diff
@@ -92,3 +93,27 @@ def test_admin(app):
...
+            data={'email': ''},
+            follow_redirects=True
+        )
+        assert b'This field is required.' in res.data
+
+        # Test user creation
+
+        res = client.post(
+            url_for("user.create_view"),
```
Comment (jirikuncar) [inline tests/test_admin.py]: `s/"/'/`

File: tests/test_admin.py [hunk 4]
```diff
@@ -92,3 +93,27 @@ def test_admin(app):
...
+            data={'email': 'test2@test.cern'},
+            follow_redirects=True
+        )
+        assert _datastore.get_user('test2@test.cern') is not None
+
+        user_data = dict(email="test3@test.cern", active=False,
```
Comment (jirikuncar) [inline tests/test_admin.py]: `s/"/'/`

File: tests/test_admin.py [hunk 5]
```diff
@@ -92,3 +93,27 @@ def test_admin(app):
...
+                         password=encrypt_password('aafaf4as5fa'))
```
Comment (jirikuncar) [inline tests/test_admin.py]: Simple possible password is preferable in tests: `123456`

File: tests/test_admin.py [hunk 6]
```diff
@@ -92,3 +93,27 @@ def test_admin(app):
...
+        _datastore.create_user(**user_data)
```
Comment (jirikuncar) [inline tests/test_admin.py]: IMHO this should be part of new test case that simulates flow defined in https://github.com/inveniosoftware/invenio-accounts/issues/154

File: tests/test_admin.py (hunk 1 above)
Comment (diegodelemos) [inline tests/test_admin.py]: Yes, I need it to make this line compatible with Python 3:
`assert b'This field is required.' in res.data`

Comment (jirikuncar) [inline tests/test_admin.py]: `b'foo'` works on Python 2.7.

File: tests/test_admin.py [hunk 7]
```diff
@@ -92,3 +93,69 @@ def test_admin(app):
             data={'rowid': -42, 'action': 'activate'},
             follow_redirects=True
         )
+
+    user_adminview['model'] = user_model
```
Comment (jirikuncar) [inline tests/test_admin.py]: why do you need it?

Comment (jirikuncar) [inline tests/test_admin.py]: Maybe line 55 and 56 should use `get` instead of `pop`. (`user_model = user_adminview.get('model')`)

Comment (diegodelemos) [inline tests/test_admin.py]: Fixed copying `user_adminview` using `dict` constructor

File: tests/test_admin.py [hunk 8]
```diff
@@ -92,3 +93,67 @@ def test_admin(app):
...
+        res = client.post(
+            url_for('user.create_view'),
+            data={'email': 'test1@test.cern'},
+            follow_redirects=True
+        )
+        assert _datastore.get_user('test1@test.cern') is not None
+
+        res = client.post(
+            url_for('user.create_view'),
+            data={'email': 'test2@test.cern'},
```
Comment (jirikuncar) [inline tests/test_admin.py]: check also `active=True` and `active=False` for test2 and test3 respectively

File: tests/test_admin.py [hunk 9]
```diff
@@ -92,3 +93,67 @@ def test_admin(app):
             data={'rowid': -42, 'action': 'activate'},
             follow_redirects=True
         )
+
+
+def test_admin_createuser(app):
+    """Test flask-admin user creation"""
+    assert isinstance(user_adminview, dict)
```
Comment (jirikuncar) [inline tests/test_admin.py]: consider creating `admin` fixture in `conftest.py` from lines 100-110 (and 44-56)

**Pull request merged**